streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -264,6 +264,72 @@ public void shouldSupportNonPrefixedProducerConfigs() {
         assertEquals(1, configs.get(ProducerConfig.METRICS_NUM_SAMPLES_CONFIG));
     }
 
+    @Test
+    public void shouldSupportCustomNonPrefixedPropertiesThatAreNotPartOfConsumerConfig() {
+        final StreamsConfig streamsConfig = new StreamsConfig(props);
+        props.put("custom.property.host", "host");
+        final Map<String, Object> consumerConfigs = streamsConfig.getConsumerConfigs("groupId", "clientId");
+        assertEquals("host", consumerConfigs.get("custom.property.host"));
```

Comment:
this seems to be equivalent to the "DUMMY" we have above -- I would suggest to remove "DUMMY" from default config as this (and below ones) add proper testing.

I think we can merge this three test and rename `shouldForwardCustomConfigsWithNoPrefixToAllClients`.

We should also add a check for `AdmitClientConfig`.

Reply:
Yes, I agree. That will make it cleaner. I'll change it.